packages/orchestration/src/exos/chain-hub.js

```diff
@@ -622,45 +622,45 @@ export const makeChainHub = (
       }
       return undefined;
     },
+
     /**
      * @param {string} partialId CAIP-10 account ID or a Cosmos bech32 address
      * @returns {AccountId}
      * @throws {Error} if chain info not found for bech32Prefix
      */
     resolveAccountId(partialId) {
-      const parsed = parseAccountId(partialId);
-      if ('namespace' in parsed) {
-        // It is already fully qualified
+      if (partialId.split(':').length === 3) {
```

Comment:
consider a comment "if it's already a CAIP-10, return it"

Reply:
done